scripts/cnode-helper-scripts/cncli.sh

```diff
@@ -522,12 +522,7 @@ cncliMigrateBlocklog() {
 cncliPTsendtip() {
   [[ ${NWMAGIC} -ne 764824073 ]] && echo "PoolTool sendtip only available on MainNet, exiting!" && exit 1
   [[ -z ${POOL_ID} || -z ${POOL_TICKER} || -z ${PT_API_KEY} ]] && echo "'POOL_ID' and/or 'POOL_TICKER' and/or 'PT_API_KEY' not set in $(basename "$0"), exiting!" && exit 1
-  
-  # Generate a temporary pooltool config
-  if ! cnode_path=$(command -v cardano-node 2>/dev/null); then
-    echo "ERROR: cardano-node not in PATH, please manually set CCLI in env file"
-    exit 1
-  fi
+  cnode_path=$(command -v ${CNODEBIN} 2>/dev/null)
```

Comment:
Ensuring CNODEBIN is in path is in env already. Hence, removed condition

Reply:
Cant we juse use ${CNODEBIN} directly in this case.
`CNODEBIN=$(command -v cardano-node)`
is already done in env.